docs/form-styling.rst

```diff
@@ -362,7 +362,6 @@ The ``style`` attribute accepts CSS-like key-value pairs for setting ``color`` a
   - sans-serif
   - monospace
   - cursive
-  - fantasy
```

Comment:
The "fantasy" font-family is rendering as wingdings for me and Szymon, so let's not recommend it.